core/src/main/java/io/questdb/griffin/engine/functions/groupby/CountDistinctIPv4GroupByFunction.java

```diff
@@ -56,7 +56,7 @@ public void clear() {
     public void computeFirst(MapValue mapValue, Record record) {
         final CompactIntHashSet set;
         if (sets.size() <= setIndex) {
-            sets.extendAndSet(setIndex, set = new CompactIntHashSet(64, 0.7, Numbers.IPv4_NULL));
+            sets.extendAndSet(setIndex, set = new CompactIntHashSet(16, 0.7, Numbers.IPv4_NULL));
```

Comment:
@puzpuzpuz could we have this  pulled out as a configuration please?

Reply:
Good idea. Done in 4e2cb4e075ffc8e7a7d58f67f926f00a25359181